### PR TITLE
stdlib/Test: fix display of `≈` with atol

### LINKED ISSUE #24919: @test with atol does not print values
Example:

```julia
julia> x = 0.9; y = 0.1;

julia> @test x ≈ y
Test Failed
  Expression: x ≈ y
   Evaluated: 0.9 ≈ 0.1
ERROR: There was an error during testing

julia> @test x ≈ y atol=0.01
Test Failed
  Expression: ≈(x, y, atol=0.01)
ERROR: There was an error during testing
```

In the second case, `x,y` are not substituted for their values.

### PR DESCRIPTION
**Pull request by iblislin**
fix #24919

### CODE REVIEW

**Comment by iblis17** [comment]
cc @cossio

File: stdlib/Test/test/runtests.jl [hunk 1]
```diff
@@ -633,22 +633,36 @@ end
...
+    src = """
+    using Test
+    x, y = 0.9, 0.1
+    @test x ≈ y atol=0.01
+    """
+    msg = read(pipeline(
+        ignorestatus(`$(Base.julia_cmd()) --startup-file=no --color=no -e $src`),
+        stderr=DevNull), String)
+
+    @test contains(msg, "Evaluated: 0.9 ≈ 0.1; atol=0.01")
```
Comment (StefanKarpinski) [inline stdlib/Test/test/runtests.jl]: Maybe put the `atol=0.01` in parens or something like that?

Comment (iblis17) [reply stdlib/Test/test/runtests.jl]: you want something like `0.9 ≈ 0.1 (atol=0.01, ...)`?

Comment (StefanKarpinski) [reply stdlib/Test/test/runtests.jl]: Yes, something like that – the semicolon just looks weird there for some reason.

Comment (iblis17) [reply stdlib/Test/test/runtests.jl]: ok, done

**Review by StefanKarpinski — APPROVED** [review]

**Pull request merged**